src/scripts/make-proposal.js

```diff
@@ -190,12 +365,54 @@ function updateApprovedTokensEncoded(proposal) {
   )
 }
 
+function sendAllEtherUsingMockWallet(proposal) {
+  const { toAddress } = proposal
+  assert(toAddress, '"toAddress" is required for sending Ether with the Mock Wallet')
+
+  console.log('    - Send all Ether to %s using mock wallet', toAddress)
+  const abi = getWalletFunctionAbi('pay')
+  return web3.eth.abi.encodeFunctionCall(
+    abi, [toAddress]
+  )
+}
+
+// function sendTokensEncoded(proposal) {
+//   const { tokenAddress, toAddress, amount } = proposal
```

Comment:
What's wrong with this one? Not working?Or nothing to use on yet?

Reply:
Hehe, nothing :)
It's just that for transferring money from the Avatar you should use the Contribution Reward, not a Erc20Token ABI. I just left it commented just in case